app/boards/arm/nice_nano/nice_nano.dtsi

```diff
@@ -44,8 +44,8 @@
 
 &i2c0 {
 	compatible = "nordic,nrf-twi";
-	sda-pin = <17>;
-	scl-pin = <20>;
+	sda-pin = <32>;
+	scl-pin = <36>;
```

Comment:
What is the reasoning for this change? This will likely break existing shield's displays. I assume this was added by accident.

Reply:
This was actually a holdover from the murphpad testing before I wrote my own definition for the tool this was being used on. This is accidental.